tests/__tests__/parser/validate.test.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+require('babel-polyfill');
+const validateAddons = require('../../../lib/utils/validate-addons');
+
+describe('Addon Validation', () => {
+	it('Should notify if the package isn\'t prefixed', async () => {
+		try {
+			await validateAddons(['webpack']);
+		} catch (object) {
+			expect(object.toString()).toEqual('TypeError: Package isn\'t a valid name\n' +
```

Comment:
Is there a reason not to use `toThrow` here?

```
expect(() => validateAddons(['webpack'])).toThrow(
			'Package isn\'t a valid name\n' +
			'It should be prefixed with \'webpack-addon\''
		);
```

works the same and is much more clear isn't it?